### DISCUSSION TITLE
unable to find möbius folder in iphone Files app

### DISCUSSION THREAD

**Discussion by aceventura10** [MobiusSync/MobiusSync#83] [General]
Hi recently I moved to syncthing and I am planning to use it in my iphone as well. Looks like möbius solved the problem of missing ios support. I installed it and paired my two devices (PC and iphone). Pairing works fine but sync fine  but when I go into Files app unable to find the folder. it is located somewhere and syncs. 

Running IOS 16 
<img src="https://user-images.githubusercontent.com/9529677/194296423-ce86de0c-5679-4ce5-81fe-92602b14d6a5.jpeg" width=50% height=50%>
<img src="https://user-images.githubusercontent.com/9529677/194296673-6f27fd7f-d1c5-47a6-ad99-cb35ba5ce161.jpeg" width=50% height=50%>

can someone help me here,

**Comment by rasa** [comment]
What happens when you click on "Open Folders"?
Also, what happens when you expand a Folder, and click "Open Folder"?
Try removing and recreating the folder, does the new folder path begin with `./`?

**Comment by aceventura10** [comment]
when I click on open folder/open folders it just goes to Files app but opens what was opened previously e.g., chrome folder. I pointed to ./ path itself. not sure what I am doing wrong. Does it has anything to do with ios 16?

<img src="https://user-images.githubusercontent.com/9529677/194363854-1766cde9-434c-474a-88a2-ac5d8bfcb9c6.jpg" width=50% height=50%>

**Comment by rasa** [comment]
Perhaps the Mobius Sync directory, in your On My iPhone folder was deleted accidentally. Try:
1. Uninstalling MobiusSync
2. Cold boot your phone
3. Reinstall MobiusSync
4. Open Files app and verify MobiusSync folder now exists
5. Add a folder share
6. Report back your success! :)

**Comment by aceventura10** [comment]
that actually did the trick, I feel stupid now, I was trying it for the last two days. thank you very much.
